### ISSUE TITLE
[Topic] Screen reader testing

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#176]
URL: https://wpaccessibility.org/docs/testing/screen-readers/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/testing/screen-readers/index.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

This content should be reviewed, expanded and restructured. 
The links need to be checked and updated.

## Resources to add

- Amanda Carson: [WordPress with a screen reader](https://acarson.wtf/2025/06/09/a-wordpress-with-a-screen-reader-updatewe-have-a-third/).
- Steve Faulker: [Screen Reader HTML Support – Lookup](https://tetralogical.github.io/screen-reader-HTML-support/lookup/lookup.html).
